policy/modules/system/systemd.fc

```diff
@@ -104,3 +104,13 @@ HOME_DIR/\.config/systemd/user(/.*)?		gen_context(system_u:object_r:systemd_unit
 /var/run/systemd/units(/.*)?		gen_context(system_u:object_r:systemd_unit_file_t,s0)
 
 /var/run/initramfs(/.*)?	<<none>>
+
+# systemd-homed
+/usr/lib/systemd/system/systemd-homed-activate\.service	--	gen_context(system_u:object_r:systemd_homed_unit_file_t,s0)
+/usr/lib/systemd/system/systemd-homed\.service		--	gen_context(system_u:object_r:systemd_homed_unit_file_t,s0)
+/usr/lib/systemd/systemd-homed				--	gen_context(system_u:object_r:systemd_homed_exec_t,s0)
+/usr/lib/systemd/systemd-homework			--	gen_context(system_u:object_r:systemd_homed_exec_t,s0)
```

Comment:
In theory you could probably put this in a separate systemd_homework_t domain as this is probably what does the actual "work". Whether that is worth to complexity I do not know and that is subjective.